modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -246,20 +247,26 @@ class NewProfileIndex extends React.Component {
     } else {
       profile = (
         <div>
-          <p>New candidate created. DCCID: {this.state.newData.candID} PSCID: {this.state.newData.pscid} </p>
-          <p><a href = {'/' + this.state.newData.candID}> Access this candidate </a></p>
+          <p>New candidate created. DCCID: {this.state.newData.CandID} PSCID: {this.state.newData.PSCID}</p>
+          <p><a href = {'/' + this.state.newData.CandID}> Access this candidate </a></p>
           <p><a href = "/new_profile/" > Recruit another candidate </a></p>
         </div>
       );
     }
-    return (<Panel title="Create a new profile">{profile}</Panel>);
+    return (
+      <div className="container col-md-5">
+        <Panel title="Create a new profile">
+          {profile}
+        </Panel>
+      </div>
+    );
   }
 }
 window.addEventListener('load', () => {
   ReactDOM.render(
     <NewProfileIndex
       dataURL = {`${loris.BaseURL}/new_profile/?format=json`}
-      submitURL = {`${loris.BaseURL}/new_profile/`}
+      submitURL = {`${loris.BaseURL}/api/v0.0.2/candidates`}
```

Comment:
The loris.BaseURL could be replaced with window.location.origin and I'm unsure why we use the BaseURL instead.

Reply:
I think it was not supported by one of the browser. (IE)

Reply:
Since we don't support IE we could probably go with `window.location.origin` going forward

Reply:
BaseURL is used throughout LORIS because `window.location.origin` will break on anything that isn't hosted under the root of the domain name (as far as I know)